db/migrate/20200512013455_add_favorites_to_pets.rb

```diff
@@ -0,0 +1,5 @@
+class AddFavoritesToPets < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :pets, :favorite, foreign_key: true
+  end
```

Comment:
I messed around a lot at the beginning, these migrations are no longer relevant. Below you'll see some REMOVE migrations for favorites